submissions/DmytryjK/task_js_dom/js/script.js

```diff
@@ -0,0 +1,186 @@
+            "Поверхня озера Морейн відображає, як в дзеркалі, десять величезних канадських піків, і це робить його одним з найкрасивіших місць в Канаді. Це одне з найбільш часто фотографованих місць в Канаді, що дуже славиться красою мальовничих місць і своїми пішохідними екскурсіями."
+        ]
+    },
+    {
+        title : "Франція",
+        subtitle : "Етрета, Нормандія",
+        imgSrc : "images/etretat normandy france.jpg",
+        imgAlt : "etretat normandy france",
+        paragraphsArray : [
+            "Étretat (Етрета) - невеликий курорт з прекрасними гальковими пляжами на узбережжі Алебастрового Ла Манша, одна з головних визначних пам'яток Нормандії, на півночі Франції. Він розташований біля самого пляжу, де води Англійського каналу сформували дивовижну красу прямовисні скелі з природними арками — один із найкрасивіших краєвидів Франції.",
+            ""
+        ]
+    },
+    {
+        title : "Гренландія",
+        subtitle : "Льодяний каньйон",
+        imgSrc : "images/Grenland canyon.jpg",
+        imgAlt : "Grenland canyon",
+        paragraphsArray : [
+            "Цей воістину дивовижний острів притягує мандрівників з усього світу, як магніт.  Аналогія банальна, але краще вигадати складно. Значна частина Гренландії справді схована під надійним щитом із льоду, який останнім часом починає активно танути. Це результат глобального потепління, на яке вже багато років намагаються звернути увагу багато громадських організацій.",
+            "У льодовиках утворюються численні каньйони - їх краса просто приголомшує. Таке враження, що природа, як найталановитіший художник на цій планеті, малює своїм пензлем хитромудрі, часом навіть сюрреалістичні картини."
+        ]
+    },
+    {
+        title : "Норвегія",
+        subtitle : "Місто - Олесунн",
+        imgSrc : "images/Alesund norway.jpg",
+        imgAlt : "Alesund norway",
+        paragraphsArray : [
+            "О́лесунн (норв. Ålesund) — місто і порт на заході Норвегії. Розташоване на північ від гирла фіорду Стур. Місто розкинулося на декількох островах, а саме — Норвьойя, Аспьойя, Гейсса (Гесса) та Окснойя, які є з'єднані мостами. За легендою, поселення бере свій початок у 9 столітті коли Ролло (Рольф) заснував поблизу маєток, де мешкав війт. Статус міста Олесунн отримав в 1848 р. Після того як місто зазнало пожежі в 1904 р., воно було відбудоване у оригінальному стилі арт-нуво.",
+            "Олесунн — регіональний торговий центр і туристична база для маршрутів в регіонах Суннмьоре, долині Норанґ, льодовиках Ойє та островах Рунде та Ґіске. В місті знаходиться найбільша риболовна гавань Норвегії, пристановище для риболовних трейлерів що виловлюють тріску та палтус з виду камбалових. Разом з Тромсьо, Олесунн є осередком ловлі арктичних тюленів."
+        ]
+    }
+];
+
+//Tabs
```

Comment:
Redundant comments - don’t use stuff ‘———————————‘ or “here I will sort my array”. The first one is ugly, second - redundant because the reviewer will understand that you do sort, from your code. The only situation in which you should use comments is when you need to describe why something is going on and this is 1% of all situations :)

Reply:
Fixed, thx)